packages/contracts/src/2.0.0/protocol/Exchange/MixinTransactions.sol

```diff
@@ -123,20 +123,27 @@ contract MixinTransactions is
         bytes32 dataHash = keccak256(data);
 
         // Assembly for more efficiently computing:
-        // keccak256(abi.encode(
+        // keccak256(abi.encodePacked(
         //     EIP712_ZEROEX_TRANSACTION_SCHEMA_HASH,
         //     salt,
-        //     signerAddress,
+        //     bytes32(signerAddress),
```

Comment:
Is this a matter of style or is there a performance difference? Both should work fine.

Reply:
Mostly style. I just think there's less cognitive overhead in "pad everything to 32 bytes" than "ABI encode". If you already know what ABI encoding is, it's pretty obvious that you can use it to accomplish this. If you don't know what ABI encoding is, it's not obvious why we are doing it at all.